src/components/network/network-table.js

```diff
@@ -159,6 +328,7 @@ const NetworkTable = (props) => {
                     props.network.twoWindingsTransformers[index]
                 }
                 columns={[
+                    makeHeaderCell('TWO_WINDING_TRANSFORMER'),
```

Comment:
WINDINGS (to be consistent with changes requested in https://github.com/gridsuite/network-modification-server/pull/10)